packages/cli/package.json

```diff
@@ -12,7 +12,7 @@
     "postbuild:esm": "rollup -c ./rollup.config.mjs",
     "build": "npm run build:cjs && npm run build:esm",
     "build-haste": "rewrite-imports -i src/ -o lib/",
-    "test": "jest"
+    "test": "NODE_ENV=test jest"
```

Comment:
Please use the `cross-env` package to set the environment variable so it can also work on Windows. :(